Classes/IndexQueue/Initializer/Page.php

```diff
@@ -112,7 +112,7 @@ protected function initializeMountPointPages(): bool
         );
 
         foreach ($mountPoints as $mountPoint) {
-            if (!$this->isMountPointValid($mountPoint)) {
+            if (!empty($mountPoint['mountPageOverlayed']) && !$this->isMountPointValid($mountPoint)) {
```

Comment:
@derMatze82 
Please move the check into the isMountPointValid method.

Reply:
@dkd-kaehm done! Sorry for force-push, merged your branch accidentally instead of rebase